src/utilities/inputProcessors.ts

```diff
@@ -15,6 +15,7 @@ export enum InputParameters {
   DisableChangelog = 'disable-changelog',
   DryRun = 'dry-run',
   NodeModule = 'node-module',
+  PackageRoot = 'pkg-root',
```

Comment:
I wonder why didn't just call it `package-root`?  I mean, nothing else is actually a short form here. :)